src/dependency.rs

```diff
@@ -82,4 +82,11 @@ pub struct Dependency {
     pub path: Option<Utf8PathBuf>,
 }
 
+impl Dependency {
+    /// The dependency's possibly renamed name.
+    pub fn renamed(&self) -> &str {
+        self.rename.as_deref().unwrap_or(&self.name)
+    }
+}
```

Comment:
`renamed` doesn't seem like the right name for this concept. It's strange that we can use a `renamed()` accessor on dependency that isn't renamed, and get anything other than `None`.